src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,24 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import { NavBar } from "../components/NavBar";
+import { SideBar } from "../components/SideBar";
+import { DashboardCard } from "../components/DashboardCard";
+import { links, sidebarLinks } from "../App";
+
+export function Dashboard() {
+  const activeLink = sidebarLinks.find(
+    (link) => window.location.pathname === link.href
+  );
+  const activeTitle = activeLink ? activeLink.text : "Dashboard";
+
+  return (
+    <div className="full-page-bg">
+      <NavBar links={links} />
+      <div className="dashboard-container">
+        <SideBar links={sidebarLinks} />
```

Comment:
I see that you still have the NavBar and SideBar in all the components. We only want to define it at the top level once, so all pages should have access to that.